pkg/api/mlflow/controller/artifact.go

```diff
@@ -26,3 +61,53 @@ func (c Controller) ListArtifacts(ctx *fiber.Ctx) error {
 	log.Debugf("artifactList response: %#v", resp)
 	return ctx.JSON(resp)
 }
+
+// GetArtifact handles `GET /artifacts/get` endpoint.
+func (c Controller) GetArtifact(ctx *fiber.Ctx) error {
+	req := request.GetArtifactRequest{}
+	if err := ctx.QueryParser(&req); err != nil {
+		return api.NewBadRequestError(err.Error())
+	}
+	log.Debugf("GetArtifact request: %#v", req)
+
+	artifact, err := c.artifactService.GetArtifact(ctx.Context(), &req)
+	if err != nil {
+		return err
+	}
+
+	filename := getFilename(req.Path)
+	ctx.Set("Content-Type", getContentType(filename))
+	ctx.Set("Content-Disposition", fmt.Sprintf("attachment; filename=%s", filename))
+	ctx.Set("X-Content-Type-Options", "nosniff")
+	ctx.Context().Response.SetBodyStreamWriter(func(w *bufio.Writer) {
+		defer artifact.Close()
+		bytesWritten, err := io.CopyBuffer(w, artifact, make([]byte, 4096))
+		if err != nil {
+			log.Errorf("error encountered in %s %s: error streaming artifact: %s", ctx.Method(), ctx.Path(), err)
+		}
+		log.Debugf("GetArtifact wrote bytes to output stream: %d", bytesWritten)
```

Comment:
you probably need `else` branch here, because otherwise `error` log message will be logged with `success` log message.

Reply:
will do